pysatNASA/instruments/timed_guvi.py

```diff
@@ -117,20 +119,33 @@ def clean(self):
 # Instrument functions
 #
 # Use the default CDAWeb and pysat methods
-
 # Set the list_files routine
-fname = 'TIMED_GUVI_L1C-2-disk-IMG_{year:04d}{day:03d}'\
-        '{hour:02d}{minute:02d}{second:02d}-{end:13d}_'\
-        'REV{rev:06d}_Av{version:02d}-{versionb:02d}r{revision:03d}.nc'
-supported_tags = {'': {'': fname}}
+fname = ''.join(('TIMED_GUVI_{res:s}-disk-{mode:s}_{{year:04d}}{{day:03d}}',
+                 '{{hour:02d}}{{minute:02d}}{{second:02d}}',
+                 '-?????????????_REV??????_',
+                 'Av{{version:02d}}-??r{{revision:03d}}.nc'))
+
+tags_fmt = { tag: fname.format(res=tag.split('_')[0].replace("2","-2").upper(),
```

Comment:
```suggestion
tags_fmt = {tag: fname.format(res=tag.split('_')[0].replace("2","-2").upper(),
```

Here and elsewhere there are whitespaces after an open bracket that need to be removed.